openwisp_controller/config/admin.py

```diff
@@ -347,6 +347,13 @@ def clean_templates(self):
                 reverse=False,
                 model=config.templates.model,
                 pk_set=templates,
+                # The template validation retrieves the device object
+                # from the database. Even if the organization of the device
+                # is changed by the user, the validation uses old
+                # organization of the device because the device is not
+                # saved yet. The raw POST data is passed here so that
+                # validations can use latest value for the device object.
```

Comment:
validations > validation

I think this last sentence can be rewritten as something like:

so that the validation can be performed using up to date data of the device object.